dm/tests/new_relay/run.sh

```diff
@@ -26,7 +26,9 @@ function test_cant_dail_upstream() {
 		"start-relay -s $SOURCE_ID1 worker1" \
 		"\"result\": true" 2
 
-	kill_dm_worker
```

Comment:
kill_dm_worker and wait_pattern_exit will waiting process exit forever, this is not good at CI, so use `check_port_offline $WORKER1_PORT 20` instead